### ISSUE TITLE
[Data@TAMU] All browse by pages except issue date

### ISSUE THREAD

**Issue by devangm** [TAMULib/data-dspace-angular#14]
**URL:** https://data-pre.library.tamu.edu/browse/title
**WCAG 2.1 AA compliant? :** No
**Violation:** 1.3.2 Meaningful Sequence
**Issues:** Search box and text order
**Notes:**  Text beneath the search box explains how to use the search box. When you navigate to the search box all that is read is "Enter the first few letters" and after you navigate out and to the next element it reads the "To browse a location in the index, enter the first few letters. For a general search, click here". I think the text should be read when user navigates to search box "To browse a location in the index, enter the first few letters".

**Comment by tamu-sad-iii** [comment]
Change link to general search text and link.

For a general search, click [here](https://data-dev.library.tamu.edu/search).

Click here for a [general search](https://data-dev.library.tamu.edu/search).
